### PR TITLE
LINKTA-341 style error and notfound pages

### PR DESCRIPTION
**Pull request by lomeintheory**
## Description

@kinskiwu Please review the changes I made. 
- Added lottie animations to both error and not-found components according to the color palette discussed.
- Styling for 'Player' has been edited to Tailwind inline styling
- Accessibility attributes have been added to lottie animations
- After reading the docs, lottie player's Control component is not necessary for our purposes and has been removed
- Not-found content has been reverted and edited to be more user friendly and less generic

Let me know if I've missed anything. Thanks!

## Type of change

- [x] ✨ New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Ran on localhost:
- To test the not-found page, I navigated to an unknown route.
- To test the error page, I threw an intentional error in the Legal component and navigated to the Legal page.
Both pages render upon encountering the related error.

## Checklist:

- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have reviewed the [**Security Best Practices Document**](SECURITY_BEST_PRACTICES.md).
- [ ] I have followed the [**Naming Conventions Guide**](NAMING_CONVENTIONS_GUIDE.md) for my changes.
- [x] I have ensured that my pull request title is descriptive.

## Additional Information:

- Any remaining styling will be handled by @ChTiSh

### CODE REVIEW

**Review by kinskiwu — DISMISSED** [review]
Please make the following changes:
- Revert the not-found content to its original
- Add lottie animation to both not-found and error.tsx according to the appropriate color pallette discussed
- Convert the styling inside 'Player' to tailwind to be consistent with the rest of the landing page 
- Add accessibility attributes to the lottie animation

Note: The `Control` component may be unnecessary for these pages. Please refer to the doc to confirm. If confirmed unnecessary pls also remove the related code. Thank you!

File: app/not-found.tsx [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 export default function NotFound() {
+  
   return (
-    <div className="flex flex-col m-auto">
-      <h2 className="font-bold flex flex-col m-auto">Error 404</h2>
-      <p>Oops! This page was not found</p>
-      <button className="border-2">
-        <Link href="/">Return Home</Link>
-      </button>
+    <div>
```
Comment (kinskiwu) [inline app/not-found.tsx]: `Page Not Found` and `We couldn't find the page you're looking for` seems duplicated. Maybe remove one of them?

File: app/not-found.tsx [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+      <p className="mt-4">We couldn&apos;t find the page you&apos;re looking for</p>
+      <div aria-label='Page not found' aria-live='assertive'>
+        <Player
+        autoplay={true}
+        loop={false}
+        src={NotFoundAnimation}
+        className="w-[350px] h-[350px]"
+        >
+        </Player>
+      </div>
```
Comment (kinskiwu) [inline app/not-found.tsx:20]: Maybe move click outside of the hyperlink text to clarify here is the word to interact with for navigation?

And adjust the styling to be consistent with error.tsx for `here` and `info@linkta.org`.

File: app/error.tsx [hunk 3]
```diff
@@ -17,17 +20,21 @@ export default function Error({ error, reset }: ErrorProps) {
...
+      <div className="mt-12 mb-12" aria-label='Error page' aria-live='assertive'>
+        <Player
+        autoplay={true}
+        loop={true}
+        className="w-[350px] h-[350px]"
+        src={ErrorAnimation}
+        >
+        </Player>
+      </div>
+      <p className="mb-48">
```
Comment (kinskiwu) [inline app/error.tsx:35]: Maybe move `click` outside of the hyperlink text to clarify `here` is the word to interact with for navigation?

Also we can add some styling, such as bold text, change color when hover, and add an underline for email etc. to make the `here` and `info@linkta.org` stand out so the user is clear where to click.

```
      <p className="mb-48">
        Please click<a href="#" onClick={() => reset()}className="font-bold hover:text-red-500"> here</a> to refresh the page or contact us at <a href="mailto:info@linkta.org" className="underline hover:text-red-500">info@linkta.org</a>
      </p>
```

File: app/error.tsx
Comment (kinskiwu) [inline app/error.tsx:1]: At the moment the animation is a bit off-centered. We can add some basic styling to align the components (I also adjusted the size of the animation from 350px to 150px for the visuals):

```
    <div className="flex flex-col items-center justify-center">
      <h2 className="mt-12 text-3xl text-left font-semibold">Oops!</h2>
      <p className="mt-4 text-left">Something went wrong!</p>
      <div className="flex justify-center items-center mt-12 mb-12" aria-label='Error page' aria-live='assertive'>
        <Player
        autoplay={true}
        loop={true}
        className="w-[150px] h-[150px]"
        src={ErrorAnimation}
        >
        </Player>
      </div>
      <p className="mb-48">
        Please click<a href="#" onClick={() => reset()}className="font-bold hover:text-red-500"> here</a> to refresh the page or contact us at <a href="mailto:info@linkta.org" className="underline hover:text-red-500">info@linkta.org</a>
      </p>
    </div>
```

image

File: app/not-found.tsx (hunk 1 above)
Comment (kinskiwu) [reply app/not-found.tsx]: Also the same comment about centering the animation.

**Review by kinskiwu — COMMENTED** [review]
I'm gonna remove this block for now in case we need to merge this PR tonight.

**Comment by lomeintheory** [comment]
@kinskiwu Just made the requested changes and ran on localhost to test how the components look. Let me know if I'm missing anything, thank you!

**Review by kinskiwu — APPROVED** [review]
Thanks for the good work!

**Review by ejwoods — APPROVED** [review]
Styling looks good 👍

**Pull request merged**